backend/business-partner-agent/src/main/java/org/hyperledger/bpa/api/PartnerAPI.java

```diff
@@ -70,4 +68,23 @@
         private JsonNode credentialData;
     }
 
+    public static PartnerAPI from(Partner p) {
+        PartnerAPI result = new PartnerAPI();
+        copyFrom(result, p);
+        return result;
+    }
+
+    public static void copyFrom(@NonNull PartnerAPI to, @NonNull Partner from) {
```

Comment:
Just as a remark and I know I introduced this type of conversion, but to reduce boilerplate code my midterm goal would be to introduce [mapstruct](https://mapstruct.org/documentation/stable/reference/html/#introduction) to the project for this kind of work.